doc/contributing/contributing.rst

```diff
@@ -31,49 +31,55 @@ To contribute to the source code of the model you will need to fork the reposito
 Quickstart Guide
 ----------------
 
-1. Fork the project on GitHub (using the fork button) and create a local clone, if you haven’t already:
+**1.** Fork the project on GitHub (using the fork button).
+
+**2.** Create a local clone (we strongly suggest keeping a separate repository for development work):
 
 ::
 
     % git clone https://github.com/user_name/MITgcm.git
 
-2. Move into your local clone directory (cd MITgcm) and and set up a remote that points to the original:
+**3.** Move into your local clone directory (cd MITgcm) and and set up a remote that points to the original:
 
 ::
 
     % git remote add upstream https://github.com/altMITgcm/MITgcm.git
 
-3. This step is optional, but highly recommended. It will ensure that your GitHub repo stays up to date with the main repository. Checkout the master branch and sync everywhere:
+**4.** Make a new branch from ``upstream/master`` (name it something appropriate, here we assume it is a new feature) and make edits on this branch:
```

Comment:
here we name the branch ``newfeature``... this connects more directly to the command below

Reply:
OK - update is on the way.